tests/Engine/Unit/Segments/SegmentModelsTest.php

```diff
@@ -59,6 +59,16 @@ public function conditionParametersTraitValues()
             [SegmentConditions::NOT_CONTAINS, 'bar', 'b', false],
             [SegmentConditions::NOT_CONTAINS, 'bar', 'bar', false],
             [SegmentConditions::NOT_CONTAINS, 'bar', 'baz', true],
+            [SegmentConditions::IN, 'foo', '', false],
+            [SegmentConditions::IN, 'ba', 'foo,bar', false],
+            [SegmentConditions::IN, 'foo', 'foo,bar', true],
+            [SegmentConditions::IN, 'bar', 'foo,bar', true],
+            [SegmentConditions::IN, 'foo', 'foo', true],
+            [SegmentConditions::IN, 1, '1,2,3,4', true],
+            [SegmentConditions::IN, 1, '', false],
+            [SegmentConditions::IN, 1, '1', true],
+            [SegmentConditions::IN, 1.5, '1.5', false],
+            [SegmentConditions::IN, false, 'false', false],
```

Comment:
We should perhaps label these 2 with a comment? Or the logic itself. It's not immediately obvious why we're not handling floats / booleans (although I fully agree with the reasoning).

Reply:
Do you think it's a good place to document this? My plan was to update the documentation with the proper explanation.

Reply:
I think both?

Reply:
Done.